recipes/wheelhouse_uploader/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "wheelhouse_uploader" %}
+{% set version = "0.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wheelhouse-uploader-{{ version }}.tar.gz
+  sha256: c59bea8f2f09f4f845765e79a47442724e869aa552443cb312695040866e66fa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-markdown
+  run:
+    # - apache-libcloud ==2.2.1
```

Comment:
This version is not available on anaconda.org. Also the package seems to be not updated for 2 years.

Reply:
I guess we'll have to do with 2.2.* as you have written below. If it's problematic we can add a branch to apache-libcloud to create 2.2.1

Reply:
For reference: https://github.com/ogrisel/wheelhouse-uploader/blob/eb32a7bb410769bb4212a9aa7fb3bfa3cef1aaec/setup.py#L32